goal_src/goal-lib.gc

```diff
@@ -546,6 +546,14 @@
   `(neq? ,thing 0)
   )
 
+(defmacro xor (a b)
+  `(not (and ,a ,b))
```

Comment:
I don't think this is right if a and b are both false.

Reply:
oops, not used. It turns out it's the exact same as `(neq? a b)` anyway.